ncm-resolver/src/main/perl/resolver.pod

```diff
@@ -41,41 +41,35 @@ NCM::resolver - NCM resolver configuration component
 Sets up the resolv.conf (and optionally the dnscache configuration).
 If dnscache is used, then dnscache will be restarted on any change.
 If DNS resolution fails after making the change, then resolv.conf
-is left in it's previous state.
-Returns
-error in case of failure.
-
-=item Unconfigure()
-
-not available.
+is left in it's previous state. Returns error in case of failure.
```

Comment:
Remove the sentence "Returns error in case of failure" because that does not make sense and a quick look at the code shows it's not true. :( In most, but not all, cases it returns 0 after an error, and >0 for success... I think the component needs some modernisation.